crates/registry/src/lib.rs

```diff
@@ -10,7 +10,8 @@
 extern crate alloc;
 
 pub use alloy_primitives::map::{DefaultHashBuilder, HashMap};
-pub use op_alloy_genesis::{ChainConfig, RollupConfig};
+pub use maili_genesis::RollupConfig;
+pub use op_alloy_genesis::ChainConfig;
```

Comment:
Ah we're leaving chain config in op-alloy got it.

Reply:
I think we might as well package this with `maili-genesis` too as discussed.

Reply:
agreed. I found a `ChainConfig` in alloy, so thought it ought to stay. but didn't find it used in `op-reth`, so don't think it's an equivalent to the alloy type, just has same name. Maybe we can rename it `L2ChainConfig`? think that would be helpful.